examples/mt-pytorch/driver.py

```diff
@@ -34,58 +24,76 @@
     print(f"Commencing server round {server_round + 1}")
 
     # Get a list of node ID's from the server
-    get_nodes_req = GetNodesRequest()
+    get_nodes_req = driver_pb2.GetNodesRequest()
 
     # ---------------------------------------------------------------------- Driver SDK
-    get_nodes_res: GetNodesResponse = driver.get_nodes(req=get_nodes_req)
+    get_nodes_res: driver_pb2.GetNodesResponse = driver.get_nodes(req=get_nodes_req)
     # ---------------------------------------------------------------------- Driver SDK
 
     # Sample three nodes
     all_node_ids: List[int] = get_nodes_res.node_ids
     print(f"Got {len(all_node_ids)} node IDs")
     num_node_ids_to_sample = 3 if len(all_node_ids) >= 3 else 1
     sampled_node_ids: List[int] = random.sample(all_node_ids, num_node_ids_to_sample)
-    print(f"Sampled {len(sampled_node_ids)} node IDs")
+    print(f"Sampled {len(sampled_node_ids)} node IDs: {sampled_node_ids}")
 
     time.sleep(sleep_time)
 
     # Schedule a task for all three nodes
     fit_ins: FitIns = FitIns(parameters=parameters, config={})
-    task = Task(task_id=123, legacy_server_message=ServerMessage(fit_ins=fit_ins))
-    task_assignment: TaskAssignment = TaskAssignment(
-        task=task, node_ids=sampled_node_ids
+    server_message = ServerMessage(fit_ins=fit_ins)
+    server_message_proto: transport_pb2.ServerMessage = serde.server_message_to_proto(
+        server_message=server_message
     )
-    create_tasks_req = CreateTasksRequest(task_assignments=[task_assignment])
+    task_ins_set: List[task_pb2.TaskIns] = []
+    for samled_node_id in sampled_node_ids:
+        new_task_ins = task_pb2.TaskIns(
+            task_id="",  # Will be created and set by the DriverAPI
```

Comment:
The `task_id` is generally required in the message itself because - after creation on the server - it will be delivered to the node that pulls the `TaskIns` from the server and works on it. It's mentioned in `driver.py` to make it more clear what's happening. It could be removed from the code example, but then the reader might wonder what to do about it when creating `TaskIns` messages.